scripts/kubernetes_test.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+set -ex
+set -o pipefail
+
```

Comment:
Please document that this assumes the existence of a local Kubernetes cluster. Might be helpful to provide the commands for starting one with Kind.